src/pal/src/misc/perftrace.cpp

```diff
@@ -691,12 +691,19 @@ static
 int
 PERFWriteCounters( pal_perf_api_info * table )
 {
-    char fileName[MAX_LONGPATH];
+    char * fileName = new char[MAX_LONGPATH];
```

Comment:
Or i can change the signature of PerfLogFileName to accept StackString and set it there.